samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java

```diff
@@ -43,6 +46,13 @@ public static void main(String[] args) {
      */
     @Override
     public BotFrameworkHttpAdapter getBotFrameworkHttpAdaptor(Configuration configuration) {
-        return new AdapterWithErrorHandler(configuration);
+        Storage storage = this.getStorage();
+        ConversationState conversationState = new ConversationState(storage);
+        UserState userState = new UserState(storage);
```

Comment:
Same as above, use the getUserState, see [here](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-integration-spring/src/main/java/com/microsoft/bot/integration/spring/BotDependencyConfiguration.java#L143). Also, we can create a different method for this, just like the conversationSate